src/test/macromanager_test.cpp

```diff
@@ -0,0 +1,85 @@
+#include "recording/macromanager.h"
+
+#include <gtest/gtest.h>
+
+#include <QDebug>
+
+#include "mixxxtest.h"
+#include "signalpathtest.h"
+
+TEST(MacrosTest, CreateMacro) {
+    auto macro = new Macro();
+    ASSERT_EQ(macro->m_length, 0);
```

Comment:
```suggestion
    auto pMacro = new Macro();
    ASSERT_EQ(pMacro->m_length, 0);
```

Also, use `pMacro->getLength()` or `pMacro->length()`. We don't need to test if the member variable has been renamed.

And don't forget to delete the object after using it, or you'll leak memory.

Reply:
Hmpf, the point of the Macro was to be a struct that is as dumb as possible and has no internals to hide. But I'll adjust it.